src/pkg/credentials/m365.go

```diff
@@ -12,6 +13,12 @@ const (
 	AzureClientSecret = "AZURE_CLIENT_SECRET"
 )
 
+// M355 creds from flags
+var (
+	AClientID     string
```

Comment:
`A` is not a sufficient prefix.  Shortening variable names like this causes more confusion than it solves.

Besides, it seems like what you're actually doing here is avoiding collision with the similarly named consts in order to export global variables.  Exporting globals vars, especially within a package whose entire purpose is to make those variables safely portable by passing them around inside a struct, produces a smell.  Is it necessary to approach the solution this way?